src/main/java/org/kohsuke/github/GHReleaseBuilder.java

```diff
@@ -103,6 +104,38 @@ public GHReleaseBuilder categoryName(String categoryName) {
         return this;
     }
 
+    public static enum MakeLatest {
+
+        /** Make this the latest release */
+        TRUE,
+        /** Do not make this the latest release */
+        FALSE,
+        /** Latest release is determined by date and higher semantic version */
+        LEGACY;
+
+        /**
+         * To string.
+         *
+         * @return the string
+         */
+        @Override
+        public String toString() {
+            return name().toLowerCase(Locale.ROOT);
+        }
+    }
+
+    /**
+     * Optional.
+     *
+     * @param latest
+     *            Whether to make this the latest release. Default is {@code TRUE}
+     * @return the gh release builder
+     */
+    public GHReleaseBuilder makeLatest(MakeLatest latest) {
```

Comment:
Unsure how you would like to test this.

A release (as returned by the GH API) does not have a `latest` property, or anything like that, to describe whether or not it is the latest release. So I don't think the existing wiremock tests are appropriate.

You, could, perhaps, add a test that verifies that if this is option is set then the appropriate field is included in the JSON request (for create and update), but a cursory inspection didn't show other tests that did that, so I haven't done that here either.